code/workspaces/web-app/src/components/stacks/StackCardActions/StackCardActions.js

```diff
@@ -96,7 +96,7 @@ export const PureStackCardActions = ({ stack, openStack, deleteStack, editStack,
           Logs
         </StackMoreMenuItem>
         <StackMoreMenuItem
-          shouldRender={editStack && ownsStack}
+          shouldRender={editStack !== undefined && ownsStack}
```

Comment:
I was kind of confused that this worked so I had a quick look at the code. As you may have seen when looking at it, I think this was happening because the `shouldRender` prop defaults to `true`. Therefore when this statement was passing through `undefined`, the `StackMoreMenuItem` was setting `shouldRender` to `true` as the default. 

I was wondering whether it was worth updating the `StackMoreMenuItem` to not have a default value for `shouldRender` as it also solves the problem, but means that you can use more conventional logical statements (like the previous `editStack && ownsStack`) when specifying whether the `StackMoreMenuItem` should render?

Reply:
I played around a bit, and went with defaulting shouldRender to false - all works a bit more like you'd expect now.

Reply:
Sounds like a sensible decision, thanks Andy!